platforms/macos/src/event.rs

```diff
@@ -44,6 +73,40 @@ impl QueuedEvent {
                     };
                 }
             }
+            Self::Announcement { text, priority } => {
+                let view = match context.view.load() {
+                    Some(view) => view,
+                    None => {
+                        return;
+                    }
+                };
+
+                let window = match view.window() {
+                    Some(window) => window,
+                    None => {
+                        return;
+                    }
+                };
+
+                let mut user_info = NSMutableDictionary::new();
+                let text = Id::into_super(NSString::from_str(&text));
+                set_object_for_key(&mut user_info, &*text, unsafe {
+                    NSAccessibilityAnnouncementKey
+                });
+                let priority = Id::into_super(Id::into_super(NSNumber::new_isize(priority)));
+                set_object_for_key(&mut user_info, &*priority, unsafe {
+                    NSAccessibilityPriorityKey
+                });
+                let user_info = Id::into_super(user_info);
```

Comment:
Nit: I'm fairly certain you can use auto-deref like this to avoid having to call `into_super` so much.

```suggestion
                let mut user_info: Id<NSMutableDictionary<NSString, NSObject>, _> = NSMutableDictionary::new();
                let text = NSString::from_str(&text);
                set_object_for_key(&mut user_info, &*text, &*unsafe {
                    NSAccessibilityAnnouncementKey
                });
                let priority = NSNumber::new_isize(priority);
                set_object_for_key(&mut user_info, &*priority, &*unsafe {
                    NSAccessibilityPriorityKey
                });
```